api/k8s/v1/k8s.proto

```diff
@@ -643,6 +643,20 @@ message CronJob {
 
   map<string, string> labels = 5;
   map<string, string> annotations = 6;
+
+  bool suspend = 7;
+  int32 active_jobs = 8;
```

Comment:
is this supposed to be repeated or is it a count? maybe `num_active_jobs` would be better in case we embed more information about the active jobs later?

Reply:
sounds good.  I'll make it num_active_jobs since its a count for now